src/Messages.js

```diff
@@ -0,0 +1,977 @@
+        id: 'cvePairStatusModal.useOverallCve',
+        description: 'Label for the use overall cve checkbox',
+        defaultMessage: 'Use overall CVE status'
+    },
+
+    cveStatusModalUpdateSuccessful: {
+        id: 'cveStatusModal.updateSuccessful',
+        description: 'Notification message in case CVEs are updated successfully',
+        defaultMessage: 'Status updated for {cveCount} CVEs'
+    },
+    cveStatusModalTitlePlural: {
+        id: 'cveStatusModal.titlePlural',
+        description: 'Title for CVE status modal with multiple CVEs',
+        defaultMessage: 'Set status for these CVEs and all systems'
+    },
+    cveStatusModalTitleSingle: {
+        id: 'cveStatusModal.titleSingle',
+        description: 'Title for CVE status modal with single CVE',
+        defaultMessage: 'Set status for this CVEs and all systems'
+    },
+    cveStatusModalOverwriteCheckbox: {
+        id: 'cveStatusModal.overwriteCheckbox',
+        description: 'label for overwrite checkbox',
+        defaultMessage: 'Do not overwrite individual system status'
+    },
+    cveStatusModalOverwriteTooltip: {
+        id: 'cveStatusModal.overwriteTooltip',
+        description: 'label for overwrite checkbox',
+        defaultMessage: `When checked, this setting does not change any pre-existing 
+        statuses set on individual systems for this CVE.`
+    },
+    cveStatusModalInfo: {
+        id: 'cveStatusModal.info',
+        description: 'label for overwrite checkbox',
+        defaultMessage: `This status is applied to all existing matching systems. 
+        Any new matching systems will have the status "Not reviewed"`
+    },
+    cveStatusModalInfoTooltip: {
+        id: 'cveStatusModal.infoTooltip',
+        description: 'label for overwrite checkbox',
+        defaultMessage: `Example: If a new system is added and matches to this vulnerability, 
+        it will be given a status "Not reviewed"`
+    },
+
+    remediationsWithAnsible: {
+        id: 'remediations.withAnsible',
+        description: 'label for remediating with Ansibel',
+        defaultMessage: 'Remediate with Ansible'
+    },
+
+    downloadExecutiveLabel: {
+        id: 'downloadExecutive.label',
+        description: 'download executive button label',
+        defaultMessage: 'Download executive report'
+    },
+
+    executiveReportTitle: {
+        id: 'executiveReport.title',
+        description: 'Executive report title',
+        defaultMessage: ` This report is an executive summary of vulnerabilities 
+            that may impact your \n Red Hat Enterprise Linux servers.`
+    },
+    executiveReportHasIdentified: {
+        id: 'executiveReport.hasIdentified',
+        description: 'Executive report title, analyzing message',
+        defaultMessage: '{ systems } and identified '
+    },
+    executiveReportRHELSystems: {
+        id: 'executiveReport.RHELSystems',
+        description: 'Executive report title substring',
+        defaultMessage: 'RHEL systems '
```

Comment:
should be `(RHEL) servers`

Reply:
it looks a bit off now 
![Screenshot from 2020-03-12 10-49-43](https://user-images.githubusercontent.com/3369346/76509973-225efb00-6451-11ea-932b-8aaaad438066.png)

Reply:
Ups, I will fix now.

Reply:
I fixed the bug, can you please check

Reply:
there is still issue 
![Screenshot from 2020-03-13 09-54-15](https://user-images.githubusercontent.com/3369346/76605406-b47a0880-6510-11ea-9a09-0faf3ebd32b8.png)
1. now the wrong text is displayed (this report is executive ...) (should be This is an executive summary...)
2. wrong breaking lines
3. there is no space between number and text, ex 1234CVEs should be 1234 CVEs
4. last but not least, in the last sentence the 10(RHEL) systems should be 10 RHEL systems ( no parenthesis here, only in the previews )

Reply:
and visually what I described 
![Screenshot from 2020-03-13 09-57-01](https://user-images.githubusercontent.com/3369346/76606160-12f3b680-6512-11ea-8682-c6807918c137.png)